src/network/MessagePublisher.js

```diff
@@ -0,0 +1,44 @@
+const exchangeName = 'fog';
+const exchangeControl = 'control';
+
+export default class MessagePublisher {
+  constructor(queue) {
+    this.queue = queue;
+  }
+
+  async sendRegister(body) {
```

Comment:
This method is sending an event to the queue: 'device.registered', maybe its name should be `sendRegisteredDevice`. I think `sendRegister` can lead to confusion when distinguishing between a command and an event.

Reply:
This observation is also applied to the other methods.